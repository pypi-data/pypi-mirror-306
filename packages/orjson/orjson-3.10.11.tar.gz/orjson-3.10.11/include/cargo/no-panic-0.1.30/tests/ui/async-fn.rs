use no_panic::no_panic;

#[no_panic]
async fn f() {
    g().await;
}

async fn g() {}

fn main() {}
