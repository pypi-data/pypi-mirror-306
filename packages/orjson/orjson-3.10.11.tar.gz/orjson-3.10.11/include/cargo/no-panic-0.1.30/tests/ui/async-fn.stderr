error: no_panic attribute on async fn is not supported
 --> tests/ui/async-fn.rs:3:1
  |
3 | #[no_panic]
  | ^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `no_panic` (in Nightly builds, run with -Z macro-backtrace for more info)
